acquia/acquia_cms
# ACMS-1049: Update to Support Drush 11

**Motivation**
Fixes #[1049](https://backlog.acquia.com/browse/ACMS-1049)

**Proposed changes**
Update Drush to 11

**Alternatives considered**
NA

**Testing steps**
- Run `composer install`
- Run `drush status` OR `drush -v`
- Check drush version should be updated to 11

**Merge requirements**
